libs/vkd3d/command.c

```diff
@@ -9882,6 +9869,43 @@ static struct d3d12_command_list *unsafe_impl_from_ID3D12CommandList(ID3D12Comma
 
 extern CONST_VTBL struct ID3D12GraphicsCommandListExtVtbl d3d12_command_list_vkd3d_ext_vtbl;
 
+static void d3d12_command_list_init_attachment_info(VkRenderingAttachmentInfoKHR *attachment_info)
+{
+    attachment_info->sType = VK_STRUCTURE_TYPE_RENDERING_ATTACHMENT_INFO_KHR;
+    attachment_info->loadOp = VK_ATTACHMENT_LOAD_OP_LOAD;
+    attachment_info->storeOp = VK_ATTACHMENT_STORE_OP_STORE;
+}
+
+static void d3d12_command_list_init_rendering_info(struct d3d12_device *device, struct vkd3d_rendering_info *rendering_info)
+{
+    unsigned int i;
+
+    rendering_info->info.sType = VK_STRUCTURE_TYPE_RENDERING_INFO_KHR;
```

Comment:
I don't see state flags being cleared out here? Is that taken care of with command list creation memset?

Reply:
Yes, we're assuming that everything is zero-initialized.